cpp/src/arrow/compute/api_vector.h

```diff
@@ -188,6 +188,29 @@ class ARROW_EXPORT PartitionNthOptions : public FunctionOptions {
   NullPlacement null_placement;
 };
 
+/// \brief Options for cumulative sum function
+class ARROW_EXPORT CumulativeSumOptions : public FunctionOptions {
+ public:
+  // Default `start` value is of type Int8 to allow type casting and compatibility rules
+  // to use the type of the function values.
```

Comment:
The default value's type doesn't match this comment

Reply:
Oops, this was an old comment. Not true nor relevant anymore.